common/model-views.cpp

```diff
@@ -2346,10 +2357,21 @@ namespace rs2
                 std::string ref_str = "N/A";
                 try
                 {
-                    auto pixel_ref = _reflectivity->get_reflectivity( noise_est, max_usable_range, ir_val );
-                    _stabilized_reflectivity.add( pixel_ref );
-                    auto stabilized_pixel_ref = _stabilized_reflectivity.get( 0.75f );
-                    ref_str = to_string() << std::dec << round( stabilized_pixel_ref * 100 ) << "%";
+                    if (_reflectivity->is_history_full())
+                    {
+                        auto pixel_ref
+                            = _reflectivity->get_reflectivity(noise_est, max_usable_range, ir_val);
+                        _stabilized_reflectivity.add(pixel_ref);
```

Comment:
Am I missing something? We only `.add(...)` when full?

Reply:
Oh, I think I get it -- we `add_depth_sample` to `_reflectivity` but only add that to the stabilized history when we have enough samples...